test/unit/org/apache/cassandra/db/streaming/ComponentManifestTest.java

```diff
@@ -42,20 +33,23 @@ public void testSerialization()
         SerializationUtils.assertSerializationCycle(expected, ComponentManifest.serializer);
     }
 
-    @Test(expected = EOFException.class)
-    public void testSerialization_FailsOnBadBytes() throws IOException
-    {
-        ByteBuffer buf = ByteBuffer.allocate(512);
-        ComponentManifest expected = new ComponentManifest(new LinkedHashMap<Component, Long>() {{ put(Component.DATA, 100L); }});
-
-        DataOutputBufferFixed out = new DataOutputBufferFixed(buf);
-
-        ComponentManifest.serializer.serialize(expected, out, MessagingService.VERSION_40);
-
-        buf.putInt(0, -100);
-
-        DataInputBuffer in = new DataInputBuffer(out.buffer(), false);
-        ComponentManifest actual = ComponentManifest.serializer.deserialize(in, MessagingService.VERSION_40);
-        assertNotEquals(expected, actual);
-    }
+    // Propose removing this test which now fails on VIntOutOfRange
```

Comment:
WFM

Reply:
@dineshjoshi  WDYT? You added this test. What was the goal?